package.json

```diff
@@ -102,7 +102,9 @@
     "prepublishOnly": "npm run test",
     "lint": "tslint -c ./config/tslint/tslint.json --project ./tsconfig.json",
     "build": "tsc && npm run lint",
+    "build:dev": "tsc -p dev.tsconfig.json && npm run lint",
     "watch": "tsc-watch --onSuccess \"npm run lint\"",
```

Comment:
ok, for now, we need one watch to debug in Chrome, because of the issue with webpack and concatenating a source path, and another to debug in VS code, but then `watch` and `watch:de` are confusing since they are both used only during development

Would be nice to fix webpack and keep only one `watch`.